R_demos/4-exomeseq-introduction.Rmd

```diff
@@ -0,0 +1,267 @@
+
+```{r load data, eval=TRUE, results='show', message=FALSE, warning=FALSE}
+
+#here() # check working directory
+entity <- synGet("syn19955828")  # download file 
+exome_data <- read_csv(entity$path) # read csv file 
+
+```
+&nbsp;
+
+# Exploring the data files
+
+The accessed file is a dataframe which contains the genomic variant data collected from normal and tumor samples of two different tumor types. The last few columns of the dataframe contain meta-data associated with the samples. All silent mutations (mutations that do not affect the sequence of amino acids in the protein product of the gene) have been removed to facilitate analysis of mutations that may affect protein sequences.
+
+## Structure of the file
+
+Lets first take a brief look at the contents of the accessed file:
+```{r explore data, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+
+##Inspect the file
+print("Contents of the file:")
+head(exome_data)
+
+```
+
+The following table contains descriptions of some of the metadata variables included in the data file:
+
+Var | Description
+--- | --- 
+ _Hugo_Symbol_  |  names of genes according to HUGO database
+ _Entrez_gene_Id_  |  Gene ID according to Entrez Database
+ _NCBI_Build_ | Reference genome that was used to align the exomeSeq data
+ _Chromosome_ | Chromosome number (range 1-22 and X,Y),  Chr M == mitochondrial genome (absent in exomes with NCBI Build == hg19)
+ _species_  |  the source of the specimen
+ _age_  |  the age of the patient 
+ _sex_  |  the sex of the patient
+ _tumorType_  |  the the type of tumor, can be one of 7 different diagnoses
+ _isCellLine_  |  indicates whether the origin tissue was a cell line or a patient
+ _study_  |  the specific initiative/consortia that the study was a part of
+ _id_ | Synapse ID of the sample (unique for each sample)
+ 
+For more information regarding the various columns in the dataset please visit here: https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/#vep
+
+&nbsp;
+
+## Metadata 
+
+The metadata in a healthcare study includes descriptions of the specimens, and the details of the studies they originated from. We can explore the metadata by visualizing various aspects of it to get a sense of the data contained within our dataset and put it in context of the disease and patients.
+
+For example, we can visualize the data distribution according to sex.
+
+```{r explore Metadata1, eval=TRUE, results='show', message=FALSE, warning=FALSE, include=TRUE}
+
+#Plot Metadata
+exome_data %>% group_by(Tumor_Sample_Barcode) %>%
+ggplot(aes(x=sex))+
+    geom_bar(aes(x=sex,fill=as.factor(tumorType)),position='dodge')+
+    scale_fill_brewer(palette="Spectral")+
+    ggtitle('Exome-seq sample distribution')
+```
+
+We can also look at the different types of mutations observed for various genes according to the tumor type. 
```

Comment:
Can you provide a link to a description of mutation types defined by VEP?

Reply:
Added the necessary info